nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -118,6 +118,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The `isPowerPC` predicate, found on `platform` attrsets (`hostPlatform`, `buildPlatform`, `targetPlatform`, etc) has been removed in order to reduce confusion.  The predicate was was defined such that it matches only the 32-bit big-endian members of the POWER/PowerPC family, despite having a name which would imply a broader set of systems.  If you were using this predicate, you can replace `foo.isPowerPC` with `(with foo; isPower && is32bit && isBigEndian)`.
 
+- `literalExample`, which has been deprecated since 21.11, has been removed.
```

Comment:
This is too fast of a removal. I think the warning should be there at least for like 2 years. And it shouldn't be removed straight away either, a `throw` should be used after the warning has been there long enough.

Reply:
> keeping the _warning_ around for another year seems a bit long.

Can we put users first? An alias has basically 0 cost.
Why not keep it around so that even the slowest moving users know what happened?